### PR TITLE
[flash_ctrl] Small fix for flash_erase_suspend test

### PR DESCRIPTION
**Pull request by nikola-miladinovic**
Fix of erase suspend test for creator and owner partition

Signed-off-by: Nikola Miladinovic <nikola.miladinovic@ensilica.com>

### CODE REVIEW

**Review by tjaychen — APPROVED** [review]
looks good to me.
we should probably wait for an approval from @eshapira as well.

**Review by matutem — APPROVED** [review]
LGTM

**Review by eshapira — APPROVED** [review]
LGTM

**Pull request merged**
